src/drivers/infiniband/MT25408_PRM.h

```diff
@@ -47,7 +47,7 @@ struct hermonprm_ud_address_vector_st {	/* Little Endian */
     pseudo_bit_t	reserved1[0x00008];
 /* -------------- */
     pseudo_bit_t	hop_limit[0x00008];    /* IPv6 hop limit */
-    pseudo_bit_t	max_stat_rate[0x00004];/* Maximum static rate control. 
+    pseudo_bit_t	max_stat_rate[0x00004];/* Maximum static rate control.
```

Comment:
Again, all the whitespace-only changes need to be removed

Reply:
My mistake, sorry.  I hadn't realised these were nicely separated out into standalone commits